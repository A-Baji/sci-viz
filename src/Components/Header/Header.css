header {
    display: flex;
    position: static;
    width: 100vw;
    height: 1in;
    background-color: #ffffff;
    box-shadow: 0px -3px 3px #C8C9C7;
    padding: 4px 0;
  }
  .header-content {
    position: static;
    height: center;
    align-self: center;
    width: 80%;
  }
  
  .header-content p {
    width: fit-content;
    font-size: 2rem;
    text-align: center;
    color: rgb(139, 243, 79);
  }
  .header-img {
    border-right: 5px solid;
    border-color: rgb(183, 247, 181);
  }