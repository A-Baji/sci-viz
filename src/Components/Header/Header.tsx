import React from 'react';
import './Header.css'
import logo from '../../logo.svg'
interface HeaderProps {
}

interface HeaderState {
}

/**
 * Header component
 */
export default class Header extends React.Component<HeaderProps, HeaderState> {  
  render() {
    return (
      <header>
        <img className='header-img' src={logo} alt='Sci-viz logo'/>
        <div className='header-content'>
          <p>DataJoint SciViz</p>
        </div>
      </header>
    );
  }
}