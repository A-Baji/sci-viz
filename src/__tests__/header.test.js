import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../Components/Header/Header.tsx';

it('Snapshot of header component', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});