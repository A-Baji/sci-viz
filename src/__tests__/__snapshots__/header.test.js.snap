// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of header component 1`] = `
<header>
  <img
    alt="Sci-viz logo"
    className="header-img"
    src=""
  />
  <div
    className="header-content"
  >
    <p>
      <i
        style={
          Object {
            "paddingLeft": "20px",
          }
        }
      >
        Powered by DataJoint
      </i>
    </p>
  </div>
</header>
`;
